dotnet-desktop-guide/net/winforms/whats-new/net100.md

```diff
@@ -2,20 +2,27 @@
 title: What's new in WinForms for .NET 10 Preview
 description: Learn about what's new in Windows Forms for .NET 10 Preview. New versions of Windows Forms are released yearly with .NET.
 ms.topic: whats-new
-ms.date: 02/25/2025
+ms.date: 03/17/2025
 #customer intent: As a developer, I want to know what's changed so that I can remain up-to-date.
 ---
 
 # What's new in Windows Forms for .NET 10 Preview
 
-This article describes what's new in Windows Forms for .NET 10 Preview 1.
+This article describes what's new in Windows Forms for for .NET 10 Preview.
 
-## Preview 1
+.NET 10 Preview 2 was released on March 18, 2025.
 
-.NET 10 Preview 1 was released on February 25, 2025.
+## Release announcements
 
-The main improvement in WinForms for .NET 10 Preview 1 was changes to clipboard serialization and deserialization.
+- [.NET 10 Preview 2](https://aka.ms/dotnet/10/preview2)
+- [.NET 10 Preview 1](https://aka.ms/dotnet/10/preview1)
 
-### Clipboard changes
+## Clipboard changes
+
+Windows Forms is shipping new code for the clipboard API. The clipboard is redesigned in a way that its code can be shared with Windows Presentation Foundation (WPF). Both desktop technologies will eventually share the same code and unify how they interact with the clipboard.
 
 .NET 9 obsoleted `BinaryFormatter`, which is used in some clipboard operations. These clipboard operations required you to opt in to compatibility package, or work around the operation. To ease the pain of moving away from `BinaryFormatter`, .NET 10 is obsoleting certain clipboard methods to indicate that they shouldn't be used. More methods are being added to help JSON serialization with clipboard data, circumventing the need for `BinaryFormatter`.
+
+## Designer improvements
```

Comment:
This heading may come off as we've made improvements in Visual Studio.  Maybe something like "More support for custom designers" ?